src/markupsafe/_speedups.c

```diff
@@ -175,30 +175,26 @@ static PyMethodDef module_methods[] = {
 	{NULL, NULL, 0, NULL}  /* Sentinel */
 };
 
+static PyModuleDef_Slot module_slots[] = {
+#ifdef Py_mod_multiple_interpreters  // Python 3.12+
+	{Py_mod_multiple_interpreters, Py_MOD_PER_INTERPRETER_GIL_SUPPORTED},
+#endif
+#ifdef Py_mod_gil  // Python 3.13+
+	{Py_mod_gil, Py_MOD_GIL_NOT_USED},
+#endif
```

Comment:
This replaces the `PyUnstable_Module_SetGIL()` call, migrating away from experimental/unstable API.